test/Libraries/NodeServicesTest/WorkspaceEventTests.cs

```diff
@@ -34,7 +34,7 @@ public void WorkspaceAddedEventIsTriggeredWhenWorkspaceIsAdded()
             OpenDynamoDefinition(@".\math\Add.dyn");
         }
 
-        [Test]
+        [Test, Category("FailureNET6")]
```

Comment:
Do you know the reason for the failure of these workspaceevent tests?

Reply:
It complains about not being on the UI thread when some of the functions in these tests are called. This is also a behavior change. I will defer this to the task where we go tough all failures.

Reply:
Are the deferred tasks filed somewhere?